src/components/DogDetail/DogDetail.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './DogDetail.css';
+
+export default function DogDetail({ dog, showDetail = false }) {
+  const getClassName = () => {
+    if (showDetail) {
+      return 'big-dog';
+    } else {
+      return '';
+    }
+  };
+
+  const getAgeString = () => {
+    if (dog.age === 0) {
+      return 'less than one';
+    } else {
+      return `${dog.age}`;
+    }
+  };
+
+  return (
+    <div className="detail">
+      <h3>Meet {dog.name}</h3>
+      <img className={getClassName()} alt="dog" src={dog.image} />
+      <p>
+        {dog.name} is a(n) {getAgeString()} year old {dog.breed}
+      </p>
+      <br />
```

Comment:
Not semantic HTML. Only a few cases where br's are considered semantic (used in poetry or mailing addresses). Consider using a p instead of span on the following element which comes with a default margin